packages/node/src/generators/application/application.ts

```diff
@@ -20,16 +20,14 @@ import {
   updateProjectConfiguration,
   updateTsConfigsToJs,
 } from '@nrwl/devkit';
-
-import { join } from 'path';
-
 import { Linter, lintProjectGenerator } from '@nrwl/linter';
-import { jestProjectGenerator } from '@nrwl/jest';
+import { jestProjectGenerator } from '@nrwl/jest/generators';
 import { runTasksInSerial } from '@nrwl/workspace/src/utilities/run-tasks-in-serial';
+import { getRelativePathToRootTsConfig } from '@nrwl/js';
+import * as shared from '@nrwl/workspace/src/utils/create-ts-config';
```

Comment:
question/suggestion: should we also move `create-ts-config` to `nrwl/js`?

Reply:
moved. move create-ts-config to packages/js/src/utils/typescript/create-ts-config.ts